### DISCUSSION TITLE
台灣治安

### DISCUSSION THREAD

**Discussion by Mitochondrionjessica** [1101-GS4719-C/Python-Community-GS4719-C#1992] [General]
最近發生兩起關注度較高的新聞，一則是中捷持刀砍人，一則是殺人未遂犯逃逸。
https://udn.com/news/story/124063/7977853

https://www.msn.com/zh-tw/entertainment/news/%E6%AE%BA%E4%BA%BA%E6%9C%AA%E9%81%82%E7%8A%AF%E9%82%84%E5%9C%A8%E9%80%83-%E8%AD%A6%E6%96%B9%E5%85%AC%E5%B8%83%E8%B6%B3%E8%B7%A1-%E8%AC%9D%E5%BF%BB%E5%9A%87%E5%82%BB-%E8%B7%9F%E6%88%91%E5%AE%8C%E5%85%A8%E9%87%8D%E7%96%8A/ar-BB1mMSuP

但相較恐怖攻擊，槍擊案好像就小巫見大巫，不知道大家對台灣的治安有甚麼看法。

**Comment by 20050201Shawn** [comment]
住台灣十九年(住過萬華跟中壢，兩個都蠻亂的地方)覺得雖然有一堆奇奇怪怪的人，治安說不上好但也不算太差。中捷隨機砍人影片一開始那個阿北跟長髮男直接上前制止，還有影片後面一群人上去一起槌嫌犯其實蠻暖心的，至少看的出來北捷事件過了十年有進步了。

**Comment by YISHIUAN0125** [comment]
雖然最近社會滿動盪的，但相較於其他國家地區，比如說美國的ohio state university好了，學校附近都還會出現學生晚上搭計程車結果失蹤的事件(發生在去年)。而且白天走在街道上都會有人吸大麻(汗)，還會有人喊有炸彈，然後拆彈小組把現場封起來之類的，
我覺得台灣治安算是不錯的。

**Comment by titan92601** [comment]
最近台中捷運的持刀砍人事件和殺人未遂犯逃逸事件引發了對台灣治安的擔憂。儘管槍擊案相對較少，這些事件仍提醒我們需要更加關注公共安全。增強警力配置、提高民眾安全意識、以及完善公共安全措施是必不可少的。我認為這些事件可能讓人對台灣的治安感到擔憂，但整體而言，台灣的治安在全球範圍內仍屬良好。確保持續的安定需要政府和民眾共同努力，才能維持良好的社會秩序和安全環境。

**Comment by tzurou** [comment]
雖然因為中捷砍人事件讓大家覺得台灣治安變差，但我個人認為這是特別個案，沒有人能從外表得知這個人是不是有潛在的危害風險，警察能做的就是第一時間趕往現場阻止事態的擴大，我們能做的就是保護好自己並在力所能及的情況下阻止情況的惡化，時常耳聞國外人口失蹤、光天化日搶劫的消息，真的覺得台灣很好了，也很慶幸自己生在治安這麼好的地方。

**Comment by EPHMR0524** [comment]
近期中捷砍人事件真的很可怕，我無法想像搭車搭到一半突然有人拿刀子在自己面前揮舞的情況，以後在車上真的不能一直回手機，要隨時注意身邊情況，不知道會不會有神經病突然拿刀子砍自己，但好家在這種事情在台灣並不常發生。

**Comment by Shang621** [comment]
台灣治安環境相對安全穩定。警方部門高效執法，保障公民安全。犯罪率較低，主要犯罪包括輕微盜竊和詐騙。街頭治安良好，人們晚間可以安心外出。然而，偶爾也有少數案件發生，如公共場所鬧事或輕微衝突。遊客應保持警覺，注意個人財物，避免夜間獨自行走，特別是在人流稀少的地區。總的來說，台灣是一個安全宜居的國家，但仍需保持警惕，並遵守當地法律法規，以確保自身安全和福祉。

**Comment by Shang621** [comment]
台灣治安環境相對安全穩定。警方部門高效執法，保障公民安全。犯罪率較低，主要犯罪包括輕微盜竊和詐騙。街頭治安良好，人們晚間可以安心外出。然而，偶爾也有少數案件發生，如公共場所鬧事或輕微衝突。遊客應保持警覺，注意個人財物，避免夜間獨自行走，特別是在人流稀少的地區。總的來說，台灣是一個安全宜居的國家，但仍需保持警惕，並遵守當地法律法規，以確保自身安全和福祉。

**Comment by buoioiry** [comment]
小時候家裡附近還會有擄人跟破車窗偷竊事件，但長大以後比較少聽到了，感覺台灣治安方面有提升不少，至少大家出門也不用太擔心東西被偷甚麼的，但捷運隨機砍人事件感覺有讓大家搭捷運時會更加注意，朋友也說最近搭車都有比較多警察所以大家都會很靠近牠們保護自身安危。

**Comment by sandyho0307** [comment]
我覺得台灣的治安在國際上算是比較好的，犯罪率相對低，大部分地區都很安全。然而，偶爾發生的暴力事件還是會讓人對治安感到憂心。這些事件提醒我們，即便是在治安相對良好的社會，也不能掉以輕心，政府和社會需要持續努力，確保公共安全。

**Comment by hbh10601** [comment]
台灣治安真的算好的了，我記得之前也在討論區看到類似的，我記得我說在歐洲真的很常被搶包包或手機等等，甚至美國搶價值多少錢以下的東西是不會被關的，原本以為高度發展國家就比較不會這樣。

**Comment by 710388** [comment]
我覺得已經算是很好了，比起其他國家至少平常半夜在路上走路不會感到太害怕

**Comment by asd99263fg** [comment]
不得不說你去過美國就知道台灣已經領先很多地方了，美國那才叫真的亂，有些州甚至根本沒有限制，甚至有一州還規定搶錢金額低於某個數字都不會犯法，相對台灣已經是真的好很多的，我覺得比起現在討論這個，不如想怎麼改善我們教育避免以後出現這樣犯罪問題

**Comment by 155Julia** [comment]
我覺得台灣治安超好的好嗎！？就是因為太少有犯案事件，所以中捷這種事情才會被新聞大肆報導，如果是在美國，這些都是稀鬆平常的雞毛小事，店家便宜的生活用品都要用玻璃門上鎖更是廣為看見。
